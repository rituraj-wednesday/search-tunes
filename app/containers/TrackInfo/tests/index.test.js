/**
 *
 * Tests for TrackInfo container
 *
 *
 */

import React from 'react';
// import { fireEvent } from '@testing-library/dom';
import { renderProvider } from '@utils/testUtils';
import { TrackInfoTest as TrackInfo } from '../index';

describe('<TrackInfo /> container tests', () => {
  it('should render and match the snapshot', () => {
    const { baseElement } = renderProvider(<TrackInfo match={{params: { trackId: 1 }}}/>);
    expect(baseElement).toMatchSnapshot();
  });
});
