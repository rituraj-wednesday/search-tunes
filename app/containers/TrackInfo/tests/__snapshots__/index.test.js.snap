// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackInfo /> container tests should render and match the snapshot 1`] = `
<body>
  <div>
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root ercpcbo5 css-14b3cuv-MuiPaper-root-MuiCard-root"
    >
      <div
        class="css-5wy8ct ercpcbo3"
      >
        <div
          class="css-8a4jlr ercpcbo1"
        >
          <button
            aria-label="Play Button"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-1oon42-MuiButtonBase-root-MuiIconButton-root"
            tabindex="0"
            type="button"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-oi4oci-MuiSvgIcon-root"
              data-testid="PlayCircleFilledRoundedIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2m-2 13.5v-7c0-.41.47-.65.8-.4l4.67 3.5c.27.2.27.6 0 .8l-4.67 3.5c-.33.25-.8.01-.8-.4"
              />
            </svg>
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <div
            class="css-vpl511 ercpcbo2"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
